### PR TITLE
Fix REPL for IE11 at new site

### PR DESCRIPTION
**Pull request by RusinovAnton**
refs: https://github.com/babel/website/issues/1627

I've added "babel-polyfill" to make REPL work as Henry suggested.

There is another error is still being thrown but it doesn't affect REPL page:
https://github.com/babel/website/issues/1627#issuecomment-391074217

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit b082d0197cb83ef6ccc7f2e72aac623c93aae95c

https://deploy-preview-1662--babel-new.netlify.com

File: .babelrc [hunk 1]
```diff
@@ -13,7 +13,7 @@
       {
         "modules": false,
         "targets": {
-          "browsers": ["last 3 versions"]
+          "browsers": ["last 3 versions", "ie >= 11"]
```
Comment (existentialism) [inline .babelrc]: IIRC, isn't `ie >=11` included in `last 3 version`?

Comment (RusinovAnton) [reply .babelrc]: Yeah, you're right actually. Silly me assumed that edge is the next version of IE. Going to fix it now

**Comment by hzoo** [comment]
We can still add `useBuiltIns: true` in preset-env for the polyfill?

**Pull request merged**